src/Doctrine/EntityClassGenerator.php

```diff
@@ -31,7 +31,7 @@ public function __construct(Generator $generator, DoctrineHelper $doctrineHelper
         $this->doctrineHelper = $doctrineHelper;
     }
 
-    public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $apiResource, bool $withPasswordUpgrade = false, bool $generateRepositoryClass = true): string
+    public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $apiResource, bool $withPasswordUpgrade = false, bool $generateRepositoryClass = true, bool $isTyped = false): string
```

Comment:
Wouldn't it be worth to create some `Config` object which holds such values instead of injecting them all over the place? It increases complexity a lot

Reply:
we should only need to check `$uses_typed_properties` in the templates with a conditional for this. see https://github.com/symfony/maker-bundle/blob/0f1d3ed2584349dc8700d7908e8a92b3742b1c99/src/Resources/skeleton/authenticator/Security52LoginFormAuthenticator.tpl.php#L25

Reply:
Thanks @OskarStark, it's not a bad idea but in my opinion, a simple ParameterBag is sufficient for 2 parameters